### ISSUE TITLE
Custom Contract descriptions/notes

### ISSUE THREAD

**Issue by Skasi** [StrangeLoopGames/EcoSuggestions#1410]
TL;DR: **Let me write descriptions for contracts.**

I like providing people with hints, like post coordinates on where to harvest the resources I want them to deliver, how much time the required task might take, etc. Something like a text field similar to the "Custom Clause", but not an actual clause. (I could misuse the custom clause option for that, but then the contract can not be auto-completed)

Example description:
_Requirements: Shovel & Calories
Please use the cart provided to deliver clay. It can transport all 95 units in one trip. You can collect clay 50m away at (1180, 60, 850), just through the tunnel - follow the signs. The task will take ~5 minutes (provided you don't accidentally flip the cart)._
